fluidreview/api.py

```diff
@@ -207,18 +209,29 @@ def parse_webhook(webhook):
         webhook.save()
 
 
-def parse_webhook_user(fluidreview_id):
+def parse_webhook_user(webhook):
     """
     Create/update User and Profile objects if necessary for a FluidReview user id
     Args:
-        fluidreview_id(int): ID of FluidReview user
+        webhook(WebhookRequest): a WebHookRequest object
     """
-    if not Profile.objects.filter(fluidreview_id=fluidreview_id):
+    klass = Klass.objects.get(klass_key=webhook.award_id)
+    profile = Profile.objects.filter(fluidreview_id=webhook.user_id).first()
+    if not profile:
         # Get user info from FluidReview API (ensures that user id is real).
-        user_info = FluidReviewAPI().get('/users/{}'.format(fluidreview_id)).json()
+        user_info = FluidReviewAPI().get('/users/{}'.format(webhook.user_id)).json()
         serializer = UserSerializer(data=user_info)
         serializer.is_valid(raise_exception=True)
-        process_user(serializer.data)
+        user = process_user(serializer.data)
+    else:
+        user = profile.user
+    if webhook.amount_to_pay:
```

Comment:
Should this be `webhook.amount_to_pay is not None`?

Reply:
✅